bottlerocket-settings-models/settings-models/Cargo.toml

```diff
@@ -8,6 +8,9 @@ publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+nvidia-device-plugin = []
```

Comment:
Probably worth a comment here as to what this enables, since `settings-models` covers a pretty wide domain.

Reply:
comment added.